css/index.css

```diff
@@ -1,3 +1,172 @@
-/* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* Desktop CSS here! */
+* {
+    box-sizing: border-box;
+}
 
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
 
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+
+body .container {
+	margin: 0 auto;
+	max-width: 880px;
+}
+
+.navigation {
+	  display: flex;
+		flex-direction: row;
+		padding-top: 15px;
```

Comment:
For your tablet view, to make the image logo be on top of the nav links, you can change the `flex-direction property` (and also remove the `position: absolute;` setting from the logo image).